.nvmrc

```diff
@@ -0,0 +1 @@
+v20.18.0
```

Comment:
Is there a reason to mandate version 20 here?

Reply:
Ah this was the original comment you were referring to (saw the `jco` one first) -- so I only put this here because this is what we were testing in CI -- I was trying to figure out which version was expected/tested for when developing `componetize-js`!

Normally I'd use `22.x` but IIRC I only found references to 20, so thought maybe there was a reason we were on that LTS release.

Reply:
Right, yeah we should probably add a node latest into the matrix.

Reply:
Yeah I'll back this out and add node latest to the matrix!